bin/live/pycbc_live_supervise_collated_trigger_fits

```diff
@@ -390,26 +389,27 @@ def combine_significance_fits(
         len(daily_files)
     )
     date_range = f'{first_date}-{end_date}'
-    outfile_name = combined_controls['outfile-format'].format(
+    outfile_name = combined_fit_controls['outfile-format'].format(
         date=day_str,
         date_range=date_range,
     )
-    combined_options['output'] = os.path.join(output_dir, outfile_name)
-    combined_options['trfits-files'] = ' '.join(daily_files)
+    combined_fit_options['output'] = os.path.join(output_dir, outfile_name)
+    combined_fit_options['trfits-files'] = ' '.join(daily_files)
 
-    combined_args = ['pycbc_live_combine_single_significance_fits']
-    combined_args += sv.dict_to_args(combined_options)
+    combined_fit_args = ['pycbc_live_combine_single_significance_fits']
+    combined_fit_args += sv.dict_to_args(combined_fit_options)
 
-    sv.run_and_error(combined_args, controls)
+    sv.run_and_error(combined_fit_args, controls)
 
-    if 'variable-significance-fits' in combined_controls:
+    if 'variable-significance-fits' in combined_fit_controls:
         logging.info("Linking to variable significance fits file")
         sv.symlink(
-            combined_options['output'],
-            combined_controls['variable-significance-fits']
+            combined_fit_options['output'],
+            combined_fit_controls['variable-significance-fits']
         )
```

Comment:
We had a conversation on one of the PyCBC calls about symlinking to local vs copying - I think this was one where copying was wanted, if that could be altered